index.ts

```diff
@@ -297,12 +317,14 @@ export const networks = getNetworks([
   dailynet_chain,
   mondaynet_chain,
   ithacanet_chain,
+  ithacanet_signer,
   jakartanet_chain,
 ])
 export const teztnets = getTeztnets([
   dailynet_chain,
   mondaynet_chain,
   ithacanet_chain,
+  ithacanet_signer,
```

Comment:
this is only used by jekyll to deploy the website and metadata. We don't want anything related to the signer setup on the website, so this line should go (as well as line 320)